Porting/Glossary

```diff
@@ -4646,13 +4646,13 @@ perl_static_inline (d_static_inline.U):
 	static _inline      (older MSVC)
 	static              (c89 compilers)
 
-PERL_SUBVERSION (Oldsyms.U):
+PERL_VERSION_PATCH (Oldsyms.U):
 	In a Perl version number such as 5.6.2, this is the 2.
 	Values greater than 50 represent potentially unstable
 	development subversions.
 	This value is manually set in patchlevel.h
 
-PERL_VERSION (Oldsyms.U):
+PERL_VERSION_MINOR (Oldsyms.U):
```

Comment:
Maybe we need a new meta-unit. Oldsyms.U sounds a little confusing now.
Given that, maybe it is a good thing to keep Oldsyms.U which documents the **old** symbols for a while (with the replaced-by names) to be removed in 7.4.0 or so.